README.md

```diff
@@ -40,3 +40,9 @@ The 4 implemented engines are:
   request before storing the source
  * TorEngine: which is backed by Tor
 
+Testing
+-------
+To run the tests install the dev dependencies listed in requirements-dev.txt,
+start the test server by running 
+`python tests/tiny_server.py`, and run the test suite with
+`nose2`.
```

Comment:
Don't forget to update this to reflect removing the actual `flask` code and the fact that we're using `py.test`.